static/img/diagrams/system-landscape.iuml

```diff
@@ -16,6 +16,7 @@ Person(cedar_team, Cedar team)
 
 System_Boundary(system, "Social Care") {
   System(social_care_cases, "Social Care Case Management", "Manage case information.", $link="https://lbhackney-it.github.io/social-care-architecture/#social-care-case-management-context-diagram")
+  System(social_care_core_pathways, "Social Care Core Pathways", "Manage assessment information.")
```

Comment:
I wonder if it would be worth specifing that this is ASC system only?

Reply:
Quite possibly, my guess is that some of this will change in future, as shortly the whole thing will be ASC only.